install.sh

```diff
@@ -16,7 +16,7 @@ else
   exit 1
 fi
 
-export CONSUL_TEMPLATE_VERSION=0.18.1
+export CONSUL_TEMPLATE_VERSION=0.19.4
```

Comment:
any concern about bumping this version at the same time as making these changes?

Reply:
No, infact, its required for the changes.